slasher/db/min_max-span_test.go

```diff
@@ -0,0 +1,118 @@
+package db
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"testing"
+
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+)
+
+type spanMapTestStruct struct {
+	validatorID uint64
+	spanMap     *ethpb.EpochSpanMap
+}
+
+var spanTests []spanMapTestStruct
+
+func init() {
+	spanTests = []spanMapTestStruct{
+		{
+			validatorID: 1,
+			spanMap: &ethpb.EpochSpanMap{
+				EpochSpanMap: map[uint64]*ethpb.MinMaxSpan{
+					1: {MinSpan: 10, MaxSpan: 20},
+					2: {MinSpan: 11, MaxSpan: 21},
+					3: {MinSpan: 12, MaxSpan: 22},
+				},
+			},
+		},
+		{
+			validatorID: 2,
+			spanMap: &ethpb.EpochSpanMap{
+				EpochSpanMap: map[uint64]*ethpb.MinMaxSpan{
+					1: {MinSpan: 10, MaxSpan: 20},
+					2: {MinSpan: 11, MaxSpan: 21},
+					3: {MinSpan: 12, MaxSpan: 22},
+				},
+			},
+		},
+		{
+			validatorID: 3,
+			spanMap: &ethpb.EpochSpanMap{
+				EpochSpanMap: map[uint64]*ethpb.MinMaxSpan{
+					1: {MinSpan: 10, MaxSpan: 20},
+					2: {MinSpan: 11, MaxSpan: 21},
+					3: {MinSpan: 12, MaxSpan: 22},
+				},
+			},
+		},
+	}
+}
+
+func TestNilDBValidatorSpanMap(t *testing.T) {
+	db := SetupSlasherDB(t)
+	defer TeardownSlasherDB(t, db)
+
+	validatorID := uint64(1)
+	vsm, err := db.ValidatorSpansMap(validatorID)
+	if err != nil {
+		t.Fatalf("nil ValidatorSpansMap should not return error: %v", err)
+	}
+	if vsm.EpochSpanMap != nil {
+		t.Fatal("ValidatorSpansMap should return nil")
+	}
+}
+
+func TestSaveValidatorSpanMap(t *testing.T) {
```

Comment:
Would recommend naming these tests with the form
```
func TestValidatorSpanMap_Save
func TestValidatorSpanMap_Delete
func TestValidatorSpanMap_NilDB
```
for better go and bazel test filtering rules

```

Reply:
done